nft_ingester/src/processors/account_based/mpl_core_processor.rs

```diff
@@ -254,12 +262,16 @@ impl MplCoreProcessor {
             created_at: account_data.slot_updated as i64,
             edition_address: None,
         });
-        models.asset_authority = Some(AssetAuthority {
-            pubkey: asset_key,
-            authority: update_authority,
-            slot_updated: account_data.slot_updated,
-            write_version: Some(account_data.write_version),
-        });
+        if let Some(upd_auth) = update_authority {
```

Comment:
this one may be None only for collections, and is not none if the asset.update_authority is none. Is this how it's intended to work?